coins/monero/src/wallet/address.rs

```diff
@@ -182,6 +182,14 @@ impl<B: AddressBytes> ToString for Address<B> {
 }
 
 impl<B: AddressBytes> Address<B> {
+  /// Generates an Address type according to the specification provided in the meta.
+  ///
+  /// WARNING: Specification on the meta are not "commands" and are
+  /// just "specifications" for the address. This function relies on the
+  /// caller to make sure all the passed parameters makes sense and assumes
+  /// specifications provided in the meta are correct and applicable for the passed in keys.
+  /// For example, passing `AddressType::Subaddress` type in the meta wont generate
+  /// a correct subaddress if the keys aren't already for a valid subaddress.
```

Comment:
"Creates an Address with the provided properties."

I don't feel a need for the warning. This isn't intended to be called from private keys, a view pair, or a scanner. It's literally creating an address object without processing. I feel there's a natural understanding it won't do anything further, especially when it cannot (subaddresses require private view key knowledge).

Users are also sheparded to the Scanner for address creation.